tests/page-objects/display/languages.wdio.page.js

```diff
@@ -0,0 +1,62 @@
+const utils = require('../../utils');
+
+const submitButton = () => $('div.form-actions button.btn.btn-primary.ng-scope');
+const addLanguageButton = () => $('i.fa.fa-plus');
```

Comment:
I was hoping we could leverage smart wdio selectors where you can find an element by its text https://webdriver.io/docs/selectors/#element-with-certain-text and have a more directed selector. 
Something like `$('span=Add new language')` might work.